src/backend/services/synthesizer.py

```diff
@@ -0,0 +1,55 @@
+from google.cloud.texttospeech import (
+    AudioConfig,
+    AudioEncoding,
+    SynthesisInput,
+    TextToSpeechClient,
+    VoiceSelectionParams,
+)
+from googleapiclient.discovery import build
+
+from backend.config import Settings
+
+
+def synthesize(text: str) -> bytes:
+    """
+    Synthesizes speech from the input text.
+
+    Args:
+        text (str): The input text to be synthesized into speech.
+
+    Returns:
+        bytes: The audio content generated from the input text in MP3 format.
+    """
+    client = TextToSpeechClient(client_options={
```

Comment:
For both `synthesize` and `detect_language`, let's add some validation checks to verify if the API key is properly set. You can throw an error here that the try/catch in the endpoint catches

Reply:
Makes sense, done